changelog/22468.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core/metrics: vault.raft_storage.bolt.write.time should be a counter not a gauge
```

Comment:
This should probably replace the old message and say "counter not a summary" right? I'm assuming we didn't release the previous change already, but maybe we did!

Reply:
No, I don't think we did. I don't think I was thinking about changelogs the right way, thanks for catching this 😅